docs/products/opensearch/howto/opensearch_alerting.rst

```diff
@@ -0,0 +1,120 @@
+-------------------
+OpenSearch alerting
+-------------------
+
+   OpenSearch alerting feature sends notifications when data from one or more indices meets certain conditions that can be customized.
+   Use case examples are such as monitoring for HTTP status code 503, CPU load average above certain percentage or watch for counts of a specific keyword in logs for a specific amount of interval,
+   notification to be configured to be sent via email, slack or webhooks and other destination.
+
+   In the following example, we are using a ``sample-host-health`` index as datasource to create a simple alert to check cpu load, action will be triggered when average of ``cpu_usage_percentage`` over ``3`` minutes is above ``75%``
+
+To create the monitor programmatically, you can skip the following manual steps to :ref:`here<Code>`
```

Comment:
Let's add structure to the document to easily see key points, then we won't need this pointer. See below suggested changes

Reply:
Removed